web/cobrands/tfl/assets.js

```diff
@@ -312,7 +311,51 @@ fixmystreet.assets.add(fixmystreet.roadworks.layer_future, {
         }
         red_routes = red_routes[0];
         return red_routes.getFeaturesWithinDistance(feature.geometry, 10).length > 0;
+    },
+    select_action: true,
+    actions: {
+        // Need to override these two from roadworks_defaults in roadworks.js
+        found: null,
+        not_found: null,
+
+        asset_found: function(feature) {
+            this.fixmystreet.actions.asset_not_found.call(this);
+            feature.layer = this;
+            var attr = feature.attributes,
+            tooltip = attr.tooltip.replace(/\\n/g, '\n'),
+            desc = attr.works_desc.replace(/\\n/g, '\n');
+
+            var $msg = $('<div class="js-roadworks-message js-roadworks-message-' + this.id + ' box-warning"></div>');
+            var $dl = $("<dl></dl>").appendTo($msg);
+            $dl.append("<dt>Location</dt>");
+            var $summary = $("<dd></dd>").appendTo($dl);
+            tooltip.split("\n").forEach(function(para) {
```

Comment:
Probably not worth it, but feels quite a bit of this could be shared with the roadworks.js in some manner.

Reply:
Agreed on both counts. I did think about factoring it together somehow with more config options but decided not to bother.